stylesheets/styles.css

```diff
@@ -0,0 +1,339 @@
+/*index.html styling*/
+
+body {
+  font-family: "Maven Pro", sans-serif;
+  padding-top: 200px;
+  max-width: 1000px;
+}
+
+img {
+  float: left;
+  margin-left: 20vw;
+  margin-bottom: 25px;
+  margin-right: 35px;
+
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+  float:left;
+  max-width: 500px;
+}
+
+#intro {
+  max-height: 200px;
+  overflow: scroll;
+}
+
+header {
+  position: fixed;
+  bottom: 0px;
+  width: 70%;
+  margin: 0 15vw;
+  background-color: #2C3D44;
+  text-align: center;
+}
+
+.white-space-bottom {
+  background-color: #FFF;
+  width: 100%;
+  height: 120px;
+}
+
+ul {
+  padding: 0;
+}
+
+li {
+  display: inline-block;
+  padding: 0;
+  margin: 0 8vw;
+}
+
+a {
+  color: #FFF;
+}
+
+footer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  margin-left: 5vw;
+}
+
+@media screen and (max-width: 950px) {
+  header {
+    width: 100%;
+    margin: 0;
+  }
+
+  body {
+    padding-top: 50px;
+  }
+
+  p {
+    margin: 0 5vw;
+  }
+
+  img {
+    float: none;
+    margin-left: 25vw;
+  }
+
+  #intro {
+    max-height: 145px;
+    margin-bottom: 45px;
+  }
+}
+
+/*about.html styling*/
+
+.Resume {
+  padding-top: 100px;
+  width: 85%;
+  margin-right: 0;
+}
+
+.resume-container {
+  max-height: 350px;
+  overflow: scroll;
+}
+
+h1, h2 {
+  font-family: "Slabo 27px", serif;
+  margin: 1.5vw 15vw;
+}
+
+h1 {
+  margin-bottom: 2vw;
+}
+
+.Resume p {
+  margin-left:18vw;
+  margin-right: 0;
+  display: inline-block;
+  float: none;
+  max-width: none;
+  margin-bottom: 10px;
+}
+
+.Resume .bold {
+  font-weight: bolder;
+  margin-right: 0;
+}
+
+.Resume .bold + p {
+  margin: 0;
+}
+
+.Resume .italics {
+  font-style: italic;
+}
+
+.Resume .italics + p {
+  margin: 0;
+}
+
+.list {
+  margin: 0 18vw;
+}
+
+@media screen and (max-width: 950px) {
+  .Resume .italics + p {
+    margin: 2vw 18vw;
+  }
+  .Resume .bold + p {
+    margin: 2vw 18vw;
+  }
+  #workplace {
+    margin: 0 18vw;
+  }
+}
+/*ortfolio.html styling*/
+
+.portfolio {
+  width: 100%;
+  margin-left: 12vw;
+  margin-right: 12vw;
+}
+
+article {
+  position: relative;
+  float: left;
+  width: 180px;
+  height: 180px;
+  padding: 20px;
+  margin-left: 19px;
+  margin-top: 20px;
+
+}
+
+.project-panel {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: 1;
+}
+
+#scrabble {
+  background-image: url("../images/scrabble_logo_small.jpg");
+  background-position: center;
+  background-size: cover;
+  opacity: 1;
+  -moz-transition: opacity 1s; /* Firefox */
+  -webkit-transition: opacity 1s; /* Safari and Chrome */
+  -o-transition: opacity 1s; /* Opera */
+  transition: opacity 1s;
+}
+
+a #scrabble:hover {
+  opacity: 0.1;
+  border: solid;
```

Comment:
adding a border on `:hover` actually shifts the content box of this a little bit, so the images shift on hover